AppLovin MAX Demo App - Java/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/max/nativead/ManualNativeLateBindingAdActivity.java

```diff
@@ -122,7 +122,7 @@ public void onShowAdClicked(View view)
         if ( nativeAd.getNativeAd() != null && nativeAd.getNativeAd().isExpired() )
         {
             // Destroy expired ad and load a new one
-            nativeAdLoader.destroy(nativeAd);
+            nativeAdLoader.destroy( nativeAd );
```

Comment:
this change came from running the auto format, command - shift - option - L

Reply:
Perfect